2018-04-19T18:36:42.9317431+01:00 0HLD63GEOELUP:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/   (ca22a1cb)
2018-04-19T18:36:43.4467431+01:00 0HLD63GEOELUP:00000001 [INF] Executing action method "Tinix.Controllers.HomeController.Index (Tinix)" with arguments ([""]) - ModelState is Valid (ba7f4ac2)
2018-04-19T18:37:09.4867431+01:00 0HLD63GEOELUP:00000001 [INF] Executing ViewResult, running view at path "/Views/Home/Index.cshtml". (9707eebe)
2018-04-19T18:37:10.9897431+01:00 0HLD63GEOELUP:00000001 [INF] Executed action "Tinix.Controllers.HomeController.Index (Tinix)" in 27588.6084ms (afa2e885)
2018-04-19T18:37:11.0447431+01:00 0HLD63GEOELUQ:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/site.css   (ca22a1cb)
2018-04-19T18:37:11.0577431+01:00 0HLD63GEOELUR:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/popper.min.js   (ca22a1cb)
2018-04-19T18:37:11.0677431+01:00 0HLD63GEOELUP:00000001 [INF] Request finished in 28082.729ms 200 text/html; charset=utf-8 (791a596a)
2018-04-19T18:37:11.0747431+01:00 0HLD63GEOELUS:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/css/bootstrap.min.css   (ca22a1cb)
2018-04-19T18:37:11.1087431+01:00 0HLD63GEOELUR:00000001 [INF] Sending file. Request path: '"/js/popper.min.js"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\js\popper.min.js"' (27b0a520)
2018-04-19T18:37:11.1137431+01:00 0HLD63GEOELUQ:00000001 [INF] Sending file. Request path: '"/css/site.css"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\css\site.css"' (27b0a520)
2018-04-19T18:37:11.1397431+01:00 0HLD63GEOELUS:00000001 [INF] Sending file. Request path: '"/css/bootstrap.min.css"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\css\bootstrap.min.css"' (27b0a520)
2018-04-19T18:37:11.1687431+01:00 0HLD63GEOELUT:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/jquery.js   (ca22a1cb)
2018-04-19T18:37:11.1767431+01:00 0HLD63GEOELUT:00000001 [INF] Sending file. Request path: '"/js/jquery.js"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\js\jquery.js"' (27b0a520)
2018-04-19T18:37:11.2147431+01:00 0HLD63GEOELUU:00000001 [INF] Request starting HTTP/1.1 GET http://localhost:5000/js/bootstrap.min.js   (ca22a1cb)
2018-04-19T18:37:11.2257431+01:00 0HLD63GEOELUQ:00000001 [INF] Request finished in 206.9651ms 200 text/css (791a596a)
2018-04-19T18:37:11.2777431+01:00 0HLD63GEOELUR:00000001 [INF] Request finished in 168.8698ms 200 application/javascript (791a596a)
2018-04-19T18:37:11.2887431+01:00 0HLD63GEOELUU:00000001 [INF] Sending file. Request path: '"/js/bootstrap.min.js"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\js\bootstrap.min.js"' (27b0a520)
2018-04-19T18:37:11.3127431+01:00 0HLD63GEOELUU:00000001 [INF] Request finished in 113.7484ms 200 application/javascript (791a596a)
2018-04-19T18:37:11.3517431+01:00 0HLD63GEOELUS:00000001 [INF] Request finished in 256.2612ms 200 text/css (791a596a)
2018-04-19T18:37:11.3797431+01:00 0HLD63GEOELUT:00000001 [INF] Request finished in 251.6987ms 200 application/javascript (791a596a)
2018-04-19T18:37:14.8917431+01:00 0HLD63GEOELUT:00000002 [INF] Request starting HTTP/1.1 GET http://localhost:5000/favicon.ico   (ca22a1cb)
2018-04-19T18:37:14.8937431+01:00 0HLD63GEOELUT:00000002 [INF] Sending file. Request path: '"/favicon.ico"'. Physical path: '"C:\Users\ondero\desktop\dev\tinix\src\Tinix\wwwroot\favicon.ico"' (27b0a520)
2018-04-19T18:37:14.8967431+01:00 0HLD63GEOELUT:00000002 [INF] Request finished in 6.2914ms 200 image/x-icon (791a596a)
2018-04-19T19:05:23.0567240+01:00 0HLD640EAE339:00000001 [ERR] An unhandled exception has occurred: One or more compilation failures occurred:
C:\Users\ondero\Desktop\dev\Tinix\src\Tinix\Views\Shared\_PostDetail.cshtml(20,30): error CS1061: 'BlogPostComment' does not contain a definition for 'comment' and no extension method 'comment' accepting a first argument of type 'BlogPostComment' could be found (are you missing a using directive or an assembly reference?) (8df55373)
Microsoft.AspNetCore.Mvc.Razor.Compilation.CompilationFailedException: One or more compilation failures occurred:
C:\Users\ondero\Desktop\dev\Tinix\src\Tinix\Views\Shared\_PostDetail.cshtml(20,30): error CS1061: 'BlogPostComment' does not contain a definition for 'comment' and no extension method 'comment' accepting a first argument of type 'BlogPostComment' could be found (are you missing a using directive or an assembly reference?)
   at Microsoft.AspNetCore.Mvc.Razor.Internal.RazorViewCompiler.CompileAndEmit(RazorCodeDocument codeDocument, String generatedCode)
   at Microsoft.AspNetCore.Mvc.Razor.Internal.RazorViewCompiler.CompileAndEmit(String relativePath)
   at Microsoft.AspNetCore.Mvc.Razor.Internal.RazorViewCompiler.CreateCacheEntry(String normalizedPath)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Razor.Internal.DefaultRazorPageFactoryProvider.CreateFactory(String relativePath)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.CreateCacheResult(HashSet`1 expirationTokens, String relativePath, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.OnCacheMiss(ViewLocationExpanderContext expanderContext, ViewLocationCacheKey cacheKey)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.LocatePageFromViewLocations(ActionContext actionContext, String pageName, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.Razor.RazorViewEngine.FindView(ActionContext context, String viewName, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.ViewEngines.CompositeViewEngine.FindView(ActionContext context, String viewName, Boolean isMainPage)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper.<RenderPartialCoreAsync>d__60.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.HtmlHelper.<PartialAsync>d__57.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperPartialExtensions.Partial(IHtmlHelper htmlHelper, String partialViewName, Object model, ViewDataDictionary viewData)
   at Microsoft.AspNetCore.Mvc.Rendering.HtmlHelperPartialExtensions.Partial(IHtmlHelper htmlHelper, String partialViewName, Object model)
   at AspNetCore._Views_Home_Detail_cshtml.<ExecuteAsync>d__0.MoveNext() in C:\Users\ondero\Desktop\dev\Tinix\src\Tinix\Views\Home\Detail.cshtml:line 5
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageCoreAsync>d__16.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderPageAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.<RenderAsync>d__14.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.<ExecuteAsync>d__21.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.ViewResult.<ExecuteResultAsync>d__26.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeResultAsync>d__19.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResultFilterAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResultExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.<Invoke>d__6.MoveNext()
